servant-server/test/Servant/ServerSpec.hs

```diff
@@ -60,6 +61,9 @@ import           Servant.Server.Internal.Router
                                             (tweakResponse, runRouter,
                                              Router, Router'(LeafRouter))
 
+-- * comprehensive api test
+
+_ = serve comprehensiveAPI (error "unused") (error "unused")
```

Comment:
A similar comment here.
Also, do we actually need two last arguments here? Can't it be just `_ = serve comprehensiveAPI`?